src/pages/Register.jsx

```diff
@@ -0,0 +1,99 @@
+import { createUserWithEmailAndPassword } from "firebase/auth";
+import React, { useState } from "react";
+import classes from "../css/Register.module.css";
+import { auth, db } from "./../firebase/firebase";
+import { setDoc, doc } from "firebase/firestore";
+import { Link, Navigate } from "react-router-dom";
+import { paths } from "../app/Route";
+
+function Register() {
+  const [email, setEmail] = useState("");
```

Comment:
Ovde imas 4 state-a koji se odnose na jednu celinu a to su register informacije. Sutra dan mozes da imas 50 inputa za npr. kreiranje neke firme npr. i ne bi trebalo da pravis 50 state-ova za svaki od parametra. 
Postoji elegantnije resenje, mozes da vidis ovde https://stackoverflow.com/questions/71663356/how-to-set-the-value-of-input-from-state-react-app